### ISSUE TITLE
Combobox: JAWS announces "black down arrow" for the arrow to open the dropdown

### ISSUE THREAD

**Issue by wkeese** [ibm-js/deliteful#612]
When you focus a Combobox **and** then open the drodown (tested on Firefox/Windows using JAWS 17), JAWS announces "black down arrow" for the arrow to open the dropdown.

**Comment by wkeese** [comment]
Relatedly, on iOS when you turn on VoiceOver, and then click the arrow, it actually "focuses" (or something similar), and iOS announces "black down arrow".

**Comment by wkeese** [comment]
I filed #615 for the VoiceOver issue.
